### ISSUE TITLE
Week10 Review

### ISSUE THREAD

**Issue by WilsonCheng-sy** [SOA-SCK/PaperDeep#38]
Hi all,

This issue summary our work in this week.
We use peer programming pattern this week, so we had multiple PR for one feature. (We would make it better next week)

#26 #27 #28 #29 #30 Add error checking and success flash message.

#25 #32 #34  Add acceptance test for the home page.

#31 Add presentation layer.

#33 Beautify our Readme.txt and other documents

#35 Add session to our website

#36 Add new feature about citation tree (will continue working in the future)

To sum up, #37 is the PR from develop to main.

**Comment by chinyakao** [comment]
Reviewed!
